src/Controller/CategoryAdminController.php

```diff
@@ -74,14 +74,14 @@ public function listAction(Request $request): Response
 
     public function treeAction(Request $request): Response
     {
-        $categoryManager = $this->get('sonata.classification.manager.category');
+        $categoryManager = $this->container->get('sonata.classification.manager.category');
```

Comment:
Do we have a test that covers this action?

Reply:
I dont know, but the get method was deprecated on 5.4, and removed on 6.0

Reply:
This is not related to this PR, but we had some problem with some admin actions. 

I'll try to add some basic tests to cover this this before releasing the stable release.